unraid_mcp/tools/info.py

```
@@ -0,0 +1,407 @@
+            counts["missing"] += 1
+        elif status == "DISK_NEW":
+            counts["new"] += 1
+        else:
+            counts["unknown"] += 1
+    return counts
+
+
+def _process_array_status(raw: dict[str, Any]) -> dict[str, Any]:
+    """Process raw array data into summary + details."""
+
+    def format_kb(k: Any) -> str:
```

Ruff (0.15.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `k`

(ANN401)